src/ICSharpCode.SharpZipLib/Zip/ZipEntry.cs

```diff
@@ -861,42 +722,29 @@ public long Crc
 		}
 
 		/// <summary>
-		/// Gets/Sets the compression method. Only Deflated and Stored are supported.
+		/// Gets/Sets the compression method. Only <see cref="CompressionMethod.Deflated">Deflated</see>
```

Comment:
Not related to this actual issue, but that's out of date - BZip2 is supported in the ZipFile case now

Reply:
Yeah, that's the problem with having this logic in `ZipEntry`. I wonder if we even should limit this here. It should be up to `ZipFile` and `ZipOutputStream` to throw on adding such an entry.

Reply:
refs #500